Edits Polkadot-JS Pt2 (#5355)
w3f/polkadot-wiki: docs/learn/learn-accounts.md

slug: ../learn-accounts
---

import RPC from "./../../components/RPC-Connection"; import Tabs from "@theme/Tabs"; import TabItem
from "@theme/TabItem"; import DocCardList from '@theme/DocCardList';

:::info User friendly wallets

> Create your Polkadot accounts with a secure and user-friendly wallet listed on the

Reviewer: ```suggestion
Create your Polkadot accounts with any of the secure and user-friendly wallet listed on the
```

[Polkadot website](https://www.polkadot.network/ecosystem/wallets/).

See the [Wallets](./wallets-index) section for more information about wallets and the
[wallets and extensions](../general/wallets-and-extensions.md) page for wallets and browser
extensions funded by the Polkadot/Kusama Treasuries or by the

> [Web3 Foundation Grants Program](../general/grants.md).

Reviewer: ```suggestion
See the [Wallets](./wallets-index) section for more information about different wallet options available, and specifically the
[wallets and extensions](../general/wallets-and-extensions.md) page, which lists the user friendly wallet projects funded by the Polkadot/Kusama Treasuries or by the
[Web3 Foundation Grants Program](../general/grants.md).
```

[cryptography page](learn-cryptography.md).

<DocCardList />

## Account Address

An address is the public part of a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }}
account. The private part is the key used to access this address. The public and private parts
together make up a {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} account. You can
think of the public address of your account, like your mailbox and the private key like the key to
open that mailbox. Anybody can send mail to your mailbox, but only you can access it as only you
have access to its key. In the context of
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} accounts, anybody can send tokens to
your public address, but only you can transact with them using your private key. That is why you
should keep your private key secret.

### Mnemonic Seed Phrase

A user's account requires a private key that can sign on to one of the
[supported curves and signature schemes](../build/build-protocol-info.md#cryptography). Without a
private key, an account cannot sign anything. In
{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} there are some exceptions of accounts

> that do not have private keys (i.e. keyless accounts). Such accounts are

Reviewer: ```suggestion
that do not have known private keys (i.e. keyless accounts). Such accounts are
```

[multi-signature accounts](./learn-account-multisig.md),
[pure proxies](./learn-proxies-pure.md#anonymous-proxy-pure-proxy), and
[system accounts](./learn-account-advanced.md#system-accounts) that are not discussed here and are
meant for an advanced audience.

A typical 12-word mnemonic seed phrase is shown below.

```

> 'caution juice atom organ advance problem want to pledge someone senior holiday very'

Reviewer: Was this a grammarly suggestion?

```suggestion
'caution juice atom organ advance problem want pledge someone senior holiday very'
```

[Advanced Account](./learn-account-advanced.md) page.

### Account Generation

Usually, there are two ways of generating a mnemonic seed:

- On a "hot" device, i.e. a device that is connected to the internet
- On a "cold" device, i.e. a device that is not (and ideally will never be) connected to the
  internet

Hot wallets are susceptible to a wide range of attacks, so it is recommended to use cold wallets
when dealing with non-trivial amounts of funds.

Generating a mnemonic seed on a browser extension or a mobile application will create a hot key or
hot wallet. Create your Polkadot accounts with a secure and user-friendly wallet listed on the
[Polkadot website](https://www.polkadot.network/ecosystem/wallets/). See also the
[Wallets](./wallets-index) section for more information about wallets and the
[wallets and extensions](../general/wallets-and-extensions.md) page for wallets and browser
extensions funded by the Polkadot/Kusama Treasuries or by the
[Web3 Foundation Grants Program](../general/grants.md).

Cold keys are generated on special devices such as those provided by [Ledger](../general/ledger.md).

> Additionally, you can generate your wallet using the [Polkadot Vault](../general/polkadot-vault.md)

Reviewer: ```suggestion
Additionally, you can generate your account using the [Polkadot Vault](../general/polkadot-vault.md)
```

mobile app (you need a dedicated air-gapped Android or iOS-compatible smartphone that you are
comfortable using only for Polkadot Vault), or a dedicated hardware implementation of Polkadot Vault
such as [the Kampela Signer](https://www.kampe.la/).

> Usually, browser extensions and mobile devices have options to import accounts generated on cold
> devices. Note that the private keys of those accounts will remain on the cold device, meaning that
> you will always need the device to sign for any transaction. Exceptions exist where you can generate
> hot [proxy accounts](./learn-proxies.md) and sign on behalf of a cold account without using the cold
> device. This is practical, especially for routine transactions.

Reviewer: ```suggestion
Usually, browser extensions and mobile devices have options to securely import accounts from cold
wallets. Note that the private keys of those accounts will remain on the cold wallet, meaning that
you will always need the device to sign any transaction. Exceptions exist where you can generate
hot wallet based [proxy accounts](./learn-proxies.md) and sign on behalf of a cold wallet account without connecting the cold device. This is practical, especially for transactions made frequently.
```

Windows or `.DS_Store` files on Mac? Those are junk; they serve no specific purpose other than
making previews a bit faster. If a folder is empty saved for such a file, you can remove the folder
to clear the junk off your hard drive. That does not mean you will lose access to this folder
forever - you can always recreate it. You have the _key_, after all - you're the computer's owner.
It just means you want to keep your computer clean until you maybe need this folder again and
recreate it. Your address is like this folder - it gets removed from the chain when nothing is in it
but gets put back when it has the existential deposit.

## Account Balance Types

In {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} there are different types of
balance depending on the account activity. Different balance types indicate whether your balance can
be used for transfers, to pay fees, or must remain frozen and unused due to an on-chain requirement.
Below is an example that displays different balance types of a Kusama account (note that the balance
types are the same for a Polkadot account).

> ![account_balance_types](../assets/account-balance-types.png)

Reviewer: I think it is okay to mention that this screenshot is from Polkadot-JS UI. As this is not part of a tutorial the user may have to follow, it should be okay.
